x-pack/plugins/discover_enhanced/ui_tests/tests/saved_search_embeddable.spec.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ScoutWorkerFixtures, expect } from '@kbn/scout';
+import { test } from '../fixtures';
+import {
+  LOGSTASH_DEFAULT_END_TIME,
+  LOGSTASH_DEFAULT_START_TIME,
+  ES_ARCHIVES,
+  KBN_ARCHIVES,
+} from '../fixtures/constants';
+
+const createSavedSearch = async (
+  kbnClient: ScoutWorkerFixtures['kbnClient'],
+  searchId: string,
+  searchTitle: string,
+  dataViewId: string
+) =>
+  await kbnClient.savedObjects.create({
+    type: 'search',
+    id: searchId,
+    overwrite: false,
+    attributes: {
+      title: searchTitle,
+      description: '',
+      columns: ['agent', 'bytes', 'clientip'],
+      sort: [['@timestamp', 'desc']],
+      kibanaSavedObjectMeta: {
+        searchSourceJSON:
+          '{"highlightAll":true,"version":true,"query":{"language":"lucene","query":""},"filter":[],"indexRefName":"kibanaSavedObjectMeta.searchSourceJSON.index"}',
+      },
+    },
+    references: [
+      {
+        id: dataViewId,
+        name: 'kibanaSavedObjectMeta.searchSourceJSON.index',
+        type: 'index-pattern',
+      },
+    ],
+  });
+
+test.describe(
+  'Discover app - saved search embeddable',
+  { tag: ['@ess', '@svlSecurity', '@svlOblt', '@svlSearch'] },
+  () => {
+    const SAVED_SEARCH_TITLE = 'TempSearch';
+    const SAVED_SEARCH_ID = '90943e30-9a47-11e8-b64d-95841ca0b247';
+    const DATA_VIEW_ID = 'logstash-*';
+    test.beforeAll(async ({ esArchiver, kbnClient, uiSettings }) => {
+      await esArchiver.loadIfNeeded(ES_ARCHIVES.LOGSTASH);
+      await kbnClient.importExport.load(KBN_ARCHIVES.DASHBOARD_DRILLDOWNS);
+      await uiSettings.set({
+        defaultIndex: 'logstash-*', // TODO: investigate why it is required for `node scripts/playwright_test.js` run
+        'timepicker:timeDefaults': `{ "from": "${LOGSTASH_DEFAULT_START_TIME}", "to": "${LOGSTASH_DEFAULT_END_TIME}"}`,
+      });
+    });
```

Comment:
It doesn't match the original test 

https://github.com/elastic/kibana/blob/3e899e748e03000eef1712dd65e01d8fc6e99930/x-pack/test/functional/apps/discover/saved_search_embeddable.ts#L24-L37

I did a change to re-use the same es_archive we use in other Discover tests (less data loading), the test logic remains the same.